routers/mapzen_results.qml

```diff
@@ -42,7 +42,7 @@ Page {
     function findRoute() {
         // Load routing results from the Python backend.
         var routePage = app.pageStack.previousPage();
-        var args = [routePage.from, gps.direction, routePage.to];
+        var args = [routePage.from, routePage.to];
```

Comment:
Yes, this occured to me. I figured the second case you present would be more common and including the heading would just lead to unnecessary confusion about routing results that seem random or inconvenient.